### ISSUE TITLE
sdf_tutorials: SDFormat extensions to URDF (the 'gazebo' tag)

### ISSUE THREAD

**Issue by azeey** [gazebosim/gazebo_test_cases#459]
## Setup
- Os: Ubuntu Jammy
- BuildType: Binary

## Links
- [Live](http://sdformat.org/tutorials?tut=sdformat_urdf_extensions)
- [Source](https://github.com/gazebosim/sdf_tutorials/blob/master/urdf/sdf_extensions.md)

## Checks
- [x] **Instructions work**
- [ ] **Images (if there are any) match the result**

---

*You can find the code used to generate this test case [here](https://github.com/audrow/yatm)*

**Comment by matosinho** [comment]
Hello!

All seems well. I tried all the examples on the page and they seem to convert to the SDF as specified. I just noticed some minor differences from the tutorial. The SDF version in the tutorial is 1.9 and when I generate the SDF is 1.11. Another one is that instead of big number like in the tutorial, such as 10000000000000000, the generated SDF on my PC was saying inf.

So I guess everything is ok. If any additional test is necessary, let me know.
